api/src/stream/stream.controller.ts

```diff
@@ -4,13 +4,36 @@ import { from, map, Observable } from 'rxjs';
 import { OrgApiParam } from 'src/types';
 import { ApiTags } from '@nestjs/swagger';
 import { StreamItem, StreamTypeEnum } from './dto/stream-item.dto';
+import { createClient } from 'redis';
 
 @Controller({
   version: '1',
 })
 @ApiTags('stream')
 export class StreamController {
-  constructor(private readonly sseSvc: StreamService) {}
+  constructor(private readonly sseSvc: StreamService) {
+    const redis = createClient({
+      url: "redis://cloudknit-redis-master.zlifecycle-system.svc.cluster.local:6379"
```

Comment:
This URL should be configurable.

Reply:
Yeah, I just hardcoded to make it work, it will be added as an env var